example/matrixMulWithMdspan/src/matrixMulMdSpan.cpp

```diff
@@ -165,17 +165,19 @@ auto example(TAccTag const&) -> int
     auto mdHostC = alpaka::experimental::getMdSpan(bufHostC);
     for(Idx i = 0; i < M; ++i)
     {
-        for(Idx j = 0; j < N; ++j)
+        if(success)
         {
-            DataType expectedValue = 0.0f;
-            for(Idx k = 0; k < K; ++k)
-            {
-                expectedValue += mdHostA(i, k) * mdHostB(k, j);
-            }
-            if(mdHostC(i, j) != expectedValue)
+            for(Idx j = 0; j < N; ++j)
             {
-                success = false;
-                break;
```

Comment:
why remove the `break` ?

Reply:
I thought it is not working, because we are in double loop and we exit only the inner loop. But this was a mistake on my side. The new implementation does the same like the old one. Maybe a little bit more inefficient.